README.md

```diff
@@ -214,6 +217,7 @@ Usage of ./k8s-gke-service-account-assigner:
       --debug                               Enable debug features
       --default-scopes string               Fallback scopes to use when annotation is not set
       --default-service-account string      Fallback service account to use when annotation is not set
+      --default-project                     If a service account without a host part (after an "@") is given, assume it refers to a user-managed service account for this project id.
```

Comment:
This doesn't match flag's help text in `main.go` (and I prefer the version in main.go: `"Fallback project id to use when service account annotation is given without an associated project id"`)

Reply:
ah, good catch